src/ui-kit/SaveButton/SaveButton.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
Make Attention to file names.
_savebutton/savebutton.js_ its not good file structure.
_savebutton/index.js_ more readable.

also make this button more useful and global.
not only for "save" pass title throw props.